package.json

```diff
@@ -16,6 +16,7 @@
     "vue-router": "^4.0.12"
   },
   "devDependencies": {
+    "@heroicons/vue": "^1.0.5",
```

Comment:
These are easier to use as SVGs in a Vue app than the icons provided by Bootstrap.